src/main/controllers/CitizenHubController.ts

```diff
@@ -189,6 +190,7 @@ export default class CitizenHubController {
       showJudgmentReceived: shouldShowJudgmentReceived(userCase, hubLinksStatuses),
       respondentResponseDeadline: userCase?.respondentResponseDeadline,
       showOrderOrRequestReceived: notifications?.length,
+      respondentIsSystemUser: checkIfRespondentIsSystemUser(userCase),
```

Comment:
I can't see a test that actually covers this - should be (ideally) both a routes test and a unit test for et.caseworker.3@hmcts.net

Reply:
Added a unit test as discussed